README.md

```diff
@@ -37,3 +37,9 @@ _Past Maintainers_
 
 - Jorge Vicente Cantero - [@jvican](https://github.com/jvican)
 - Ólafur Páll Geirsson - [@olafurpg](https://github.com/olafurpg)
+
+## Protocol Changelog
+
+### Unreleased
+
+- Add unstable `run/printStdout`, `run/printStderr` and `run/readStdin` notifications
```

Comment:
What do you mean by unstable? What is the process to make them stable?

Reply:
We introduced `unstable` additions to the protocol for new changes that haven't been tested yet on implementations. The expectation is that they may change in incompatible ways even in minor versions. We mark them stable when there is consensus that they work as they are in real world implementations

Reply:
Unstable means that they are exempt from semver stability guarantee. The process to make them stable is to wait some time, see if we need to introduce some changes and remove the @unstable annotation